src/hacbs/components/ImportForm/useImportSteps.tsx

```diff
@@ -58,6 +61,25 @@ export const buildStep = (): FormikWizardStep => ({
   canJumpTo: false,
   disableBack: true,
   cancelButtonText: 'Go to application',
+  nextButtonText: 'Next',
+});
+
+export const integrationTestStep = (): FormikWizardStep => ({
+  id: 'integraiontest',
+  name: 'Add integration test',
+  component: <IntegrationTestSection />,
+  canJumpTo: false,
+  nextButtonText: 'Next',
+  validationSchema: integrationTestValidationSchema,
+  cancelButtonText: 'Go to application',
+  onSubmit: createAppIntegrationTest,
+});
+
+export const environmentStep = (): FormikWizardStep => ({
+  id: 'environment',
+  name: 'Manage Environment',
+  component: <></>,
+  canJumpTo: false,
   nextButtonText: 'Done! Go to app',
 });
```

Comment:
Is this empty step still necessary to include right now? 
If so, add `cancelButtonText: 'Go to application',`

Reply:
Updated